tensorboard/plugins/interactive_inference/README.md

```diff
@@ -57,6 +57,11 @@ You can use the What-If Tool to analyze a classification or regression
 that takes TensorFlow Example or SequenceExample protos
 (data points) as inputs directly in a jupyter or colab notebook.
 
+You can also use What-If-Tool with your own custom prediction function that takes
+Tensorflow Example and produces predictions. In this mode, you can load any model
```

Comment:
I would say "takes TensorFlow examples and produces predictions" (that way it includes Examples and SequenceExamples)

Reply:
done!